sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/ConfigurationManagerTests.cs

```diff
@@ -0,0 +1,137 @@
+                TenantId = tenantId,
+                AudienceAppId = audienceAppId,
+                TokenIssuerV1 = issuerv1,
+                TokenIssuerV2 = issuerv2,
+                OidcMetadataUrl = null
+            };
+            ConfigurationManager configurationManager = null;
+            Assert.DoesNotThrow(() => configurationManager = new ConfigurationManager(attribute), "Test when attribute's OidcMetadataUrl is null");
+            Assert.IsNotNull(configurationManager, "Test when attribute's OidcMetadataUrl is null");
+            Assert.IsTrue(configurationManager.ConfiguredService.IsDefault, "When full configuration not provided, default service should be set");
+        }
+
+        [Test]
+        public void Test_Success_Custom()
+        {
+            var attribute = new AuthenticationEventsTriggerAttribute
+            {
+                TenantId = tenantId,
+                AudienceAppId = audienceAppId,
+                TokenIssuerV1 = issuerv1,
+                TokenIssuerV2 = issuerv2,
+                OidcMetadataUrl = oidcHost
+            };
+            ConfigurationManager configurationManager = null;
+            Assert.DoesNotThrow(() => configurationManager = new ConfigurationManager(attribute), "Test when attribute's OidcMetadataUrl is null");
+            Assert.IsNotNull(configurationManager, "Test when attribute's OidcMetadataUrl is null");
+            Assert.IsFalse(configurationManager.ConfiguredService.IsDefault, "When full configuration not provided, default service should be set");
+
+            Assert.AreEqual(tenantId, configurationManager.ConfiguredService.TenantId, "TenantId should be set to the custom");
+            Assert.AreEqual(audienceAppId, configurationManager.ConfiguredService.ApplicationId, "AudienceAppId should be set to the custom");
+            Assert.AreEqual(oidcHost, configurationManager.ConfiguredService.OidcMetadataUrl, "OidcMetadataUrl should be set to the custom");
+            Assert.AreEqual(issuerv1, configurationManager.ConfiguredService.TokenIssuerV1, "TokenIssuerV1 should be set to the custom");
+            Assert.AreEqual(issuerv2, configurationManager.ConfiguredService.TokenIssuerV2, "TokenIssuerV2 should be set to the custom");
+        }
+
+        [Test]
+        public void Test_Success_Ests()
+        {
+            var attribute = new AuthenticationEventsTriggerAttribute
+            {
+                TenantId = tenantId,
+                AudienceAppId = audienceAppId
+            };
+            ConfigurationManager configurationManager = null;
+            Assert.DoesNotThrow(() => configurationManager = new ConfigurationManager(attribute), "Test when attribute's OidcMetadataUrl is null");
+            Assert.IsNotNull(configurationManager, "Test when attribute's OidcMetadataUrl is null");
+            Assert.IsTrue(configurationManager.ConfiguredService.IsDefault, "When full configuration not provided, default service should be set");
+        }
```

Comment:
2 more tests to verify: 
Oidc is present, but TokenIssuerV1 and TokenIssuerV2 are not. 
TokenIssuerV1 and TokenIssuerV2 are present, but Oidc is not.

Reply:
let me add those

Reply:
`Oidc is present, but TokenIssuerV1 and TokenIssuerV2 are not.`:
* added Test_When_Attributes_TokenIssuerV1_and_TokenIssuerV2_Is_Null
`TokenIssuerV1 and TokenIssuerV2 are present, but Oidc is not.`
* already covered by Test_When_Attributes_OpenIdMetadataUrl_Is_Null